### DISCUSSION TITLE
Some assistance with Feature request and related PR

### DISCUSSION THREAD

**Discussion by mdebarros** [argoproj/argo-cd#18235] [General]
Greetings,

I have opened both a Feature request and the associated PR following the contributor guides (as far as I am aware) as follows:
- Issue: https://github.com/argoproj/argo-cd/issues/17003
- PR: https://github.com/argoproj/argo-cd/pull/17004

However, I have not had any feedback on either.

How can I go about doing so?

Kind Regards,
Miguel

**Comment by rumstead** [comment]
A lot of PRs to go through, I bet the maintainers are just overwhelmed. Have you tried using the custom health checks? It's a more scalable way for folks to BYO health checks without PRing into Argo.

https://argo-cd.readthedocs.io/en/stable/operator-manual/health/#custom-health-checks

**Reply by ChristianCiach** [reply]
[The documentation encourages contributing health checks to the project](https://argo-cd.readthedocs.io/en/stable/operator-manual/health/#way-2-contribute-a-custom-health-check), so it's a bit disconcerting when PR are left without a reaction from any maintainer for months. 

I mean absolutely no disrespect to the maintainers, but letting PRs pile up without any kind of feedback is usually a sign of an unhealthy project. And in this case it actively discourages me from contributing at all.

**Reply by rumstead** [reply]
For the PR I would suggest posting in slack. It’s pretty active with maintainers. 

Regarding the health of the project, I wouldn’t call it unhealthy. I would call it getting swarmed in popularity due to its success and not having enough people giving back. If you look at other projects, I would consider healthy, you will see similarities, eg kubernetes, go. You should get involved!

**Reply by ChristianCiach** [reply]
Yes, I regret using the word "unhealthy". You are right that kubernetes and related projects (sigs) suffer the same issue, and it discourages me to contribute to them just as well.

**Comment by mdebarros** [comment]
> For the PR I would suggest posting in slack. It’s pretty active with maintainers.
> 
> Regarding the health of the project, I wouldn’t call it unhealthy. I would call it getting swarmed in popularity due to its success and not having enough people giving back. If you look at other projects, I would consider healthy, you will see similarities, eg kubernetes, go. You should get involved!

Thanks for the response.

I will give slack a try.
